### DISCUSSION TITLE
Show dublicates

### DISCUSSION THREAD

**Discussion by w15eacre** [keepassxreboot/keepassxc#11613] [Ideas]
I found this topic: https://github.com/keepassxreboot/keepassxc/discussions/8420
It's not convenient to find duplicate entries. Why not group these entries by login and URL? Another feature could be merging records.

**Comment by droidmonkey** [comment]
These are existing issues/requests in consideration. It's a non-trivial task because your definition of a duplicate is likely very different from another person's. Also, how "close" should they be to be considered the same? Can I mark entries as "known duplicates" so they don't appear again in a search? It's not as easy as a simple compare function. 

Merging two selected entries is equally challenging. You'll need to make a merge gui to indicate which data wins out from each entry.

**Comment by w15eacre** [comment]
I offered the easiest way to check entries, but it’s not good enough. The URL needs to be parsed for analysis, comparing step by step: schema, domain, and port. In my opinion, this solution will work for the majority of entries.

Yes, a user must unmark duplicates if they aren’t actual duplicates. The merging feature isn’t directly related to the duplication feature and may be implemented later.

The first step is to define what is considered a duplicate.
